src/frequenz/sdk/timeseries/__init__.py

```diff
@@ -5,17 +5,22 @@
 Handling of timeseries streams.
 
 A timeseries is a stream (normally an async iterator) of
-[samples][frequenz.sdk.timeseries.Sample].
+[`Sample`][frequenz.sdk.timeseries.Sample]s.
```

Comment:
Why `s` at the end?

Reply:
Is just regular text, so the sentence reads "A timeseries is a stream (normally an async iterator) of `Sample`s ..." (instead of "A timeseries is a stream (normally an async iterator) of `Sample` ...". I just wanted to use the class name because I thought it was more clear, but I'm not sure anymore, I could leave it as it was too.